chain/chain/src/runtime/tests.rs

```diff
@@ -788,13 +788,28 @@ fn test_get_validator_info() {
     let staking_transaction = stake(1, &signer, &block_producers[0], 0);
     let mut expected_blocks = [0, 0];
     let mut expected_chunks = [0, 0];
+    let mut expected_endorsements = [0, 0];
     let update_validator_stats =
-        |env: &mut TestEnv, expected_blocks: &mut [u64], expected_chunks: &mut [u64]| {
+        |env: &mut TestEnv,
+         expected_blocks: &mut [u64],
+         expected_chunks: &mut [u64],
+         expected_endorsements: &mut [u64]| {
```

Comment:
Because we touch this code, can we replace &[u64] with &[u64; 2]? It would make it much easier to read.

Reply:
Done in https://github.com/near/nearcore/pull/11121/commits/fcc4e9ad0c6f9de7aa9d2ce431ab10d2067d4fc4